Input/InputField.js

```diff
@@ -265,10 +265,7 @@ const InputFieldBase = kind({
 					}
```

Comment:
Can we replace line 252 with `forwardCustomWithPrevent`?

Reply:
✅